config/dev.toml

```diff
@@ -16,7 +16,7 @@ read_timeout = 5000000000
 write_timeout = 5000000000
 shutdown_timeout = 5000000000
 
-log_location = "log"
+log_location = ""
```

Comment:
dev is a local build and we have a log folder where this is works and useful - why remove?

Reply:
I found it more useful to log to stdout instead of having to tail a new log location each time.

Reply:
is it possible to do both with an additional setting instead of removing this?

Reply:
it's been valuable to view logs even after the server dies and do comparisons

Reply:
this could work https://gin-gonic.com/docs/examples/write-log/

Reply:
Created https://github.com/TBD54566975/ssi-service/issues/462